Bottom: 8392b73e3e5fb691fd3bb88919f19ffbae7e0503
Top:    c0d6266400957e0781dfdb8ea52d7b21165d20f1
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-29 09:24:23 -0500

Refresh of add-buildbot-role

---
diff --git a/roles/buildbot-master/tasks/main.yml b/roles/buildbot-master/tasks/main.yml
index 60a54247..84a40408 100644
--- a/roles/buildbot-master/tasks/main.yml
+++ b/roles/buildbot-master/tasks/main.yml
@@ -17,19 +17,18 @@
     group: '{{ buildbot_uid }}'
     owner: '{{ buildbot_uid }}'
   loop:
-    - buildbot.tac
     - master.cfg
     - openembedded.py
     - ptxdist.py
 
 - name: Populate config templates
-  copy:
-    src: "{{ item }}"
+  template:
+    src: "{{ item }}.j2"
     dest: "{{ config_hostdir }}/{{ item }}"
     group: '{{ buildbot_uid }}'
     owner: '{{ buildbot_uid }}'
   loop:
-    - buildbot.tac.j2
+    - buildbot.tac
 
 - name: Checkout image repo
   git:
